Scala/Scala.sublime-syntax

```diff
@@ -89,7 +90,7 @@ contexts:
       captures:
         1: keyword.declaration.scala
         2: entity.name.function.declaration
-    - match: '(case)?\b(class|trait|object)\s+([^\s\{\(\[]+)'
+    - match: '\b(case\s+)?(class|trait|object)\s+([^\s\{\(\[]+)'
```

Comment:
You could avoid assigning the keyword scope to whitespace by adding a non-capturing group: `(?:(case)\s+)?`